src/__stories__/no-asset-feedback-screen-story.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
```

Comment:
instead of no-asset-feedback-screen-story, I'd create a diferent story file for every component, one for InfoFeedbackScreen and one for ErrorFeedbackScreen